docs/api/ref/GetAgent.md

```diff
@@ -0,0 +1,253 @@
+---
+id: GetAgent
+sidebar_label: GetAgent
+title: GetAgent
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The `getAgent()` function is the recommended way for **web applications** to connect to an FDC3 Desktop Agent. The function is exported by the [`@finos/fdc3`](https://www.npmjs.com/package/@finos/fdc3) NPM module and returns a [Desktop Agent](./DesktopAgent) implementation:
+
+<Tabs groupId="lang">
+<TabItem value="ts" label="TypeScript">
+
+```ts
+import { getAgent, DesktopAgent } from "@finos/fdc3";
+
+try {
+    const fdc3: DesktopAgent = await getAgent();
+    //do FDC3 things here
+} catch (e) {
```

Comment:
this isn't consistent with the suggestion from supported-platforms.md, which used : AgentError?

Reply:
@robmoffat Do you mean it should type the error, or something more?

```suggestion
} catch (e: AgentError) {
```